peer/store-utils.go

```diff
@@ -70,10 +70,18 @@ func GetPeers() ([]Peer, error) {
 			}).Error("Failed to unmarshal peer")
 			continue
 		}
+		if p.ID.String() == context.MyUUID.String() {
+			// Skip self's information
+			continue
+		}
 		peers[i] = p
+		i = i + 1
+	}
+	if i == 0 {
```

Comment:
This is intended. If we use i defined in the for loop then it gets automatically incremented after every iteration and even a `continue` statement would increment it which we don't want since we want to return the peer structure populated with all the peers but the self.